actions/generate.action.ts

```diff
@@ -44,10 +45,19 @@ const generateFiles = async (inputs: Input[]) => {
   const collection: AbstractCollection = CollectionFactory.create(
     collectionOption || configuration.collection || Collection.NESTJS,
   );
+  
+  const caseType = (
+      configuration?.generateOptions?.caseNaming
+      || 'kebab-or-snake'
```

Comment:
Should we document that `kebab-or-snake` will be the default option when caseNaming is not specified?